pkg/meta/meta.go

```diff
@@ -50,6 +56,21 @@ func (m *Meta) GetString(key string) (string, error) {
 	return v.AsString()
 }
 
+// GetEncryptedString decorates GetString and decrypt its output with the given symmetric encryption key.
+func (m *Meta) GetEncryptedString(key string, encryptionKey []byte) (string, error) {
+	v, err := m.GetString(key)
```

Comment:
get the bytes directly instead of casting below?

Reply:
If I change it to `v, err := m.GetBytes(key)` I get: 

```
func called on wrong kind: "AsBytes" called on a string node (kind: string), but only makes sense on bytes
```

Reply:
you should not cast there: https://github.com/ucan-wg/go-ucan/pull/51/files#diff-898270f422155d6d5b46facaf0aaafe21c07e22a7ee4544b2c5ae0f7ed9eb3c6R164